boardswarm-cli/src/main.rs

```diff
@@ -389,61 +408,71 @@ struct ConfigureArg {
 
 #[derive(Debug, Subcommand)]
 enum Command {
-    /// Configure boardswarm instances
+    /// Configure boardswarm instances access
     Configure(ConfigureArg),
     /// Retrieve the login information from the remote boardswarm server
     LoginInfo,
     /// Actuator specific commands
     Actuator {
+        /// Which actuator to use
         actuator: u64,
         #[command(subcommand)]
         command: ActuatorCommand,
     },
     /// Console specific commands
     Console {
+        /// Which console to use
         console: u64,
         #[command(subcommand)]
         command: ConsoleCommand,
     },
     /// Volumes specific commands
     Volume {
+        /// Which volume to use
         volume: u64,
         #[command(subcommand)]
         command: VolumeCommand,
     },
     /// Device specific commands
     Device {
         #[arg(value_parser = parse_device)]
+        /// Which device to use
         device: DeviceArg,
         #[command(subcommand)]
         command: DeviceCommand,
     },
     /// Commands specific to rockchip devices
     Rock {
         #[arg(value_parser = parse_device)]
+        /// Which device to use
         device: DeviceArg,
         #[command(subcommand)]
         command: RockCommand,
     },
     /// List all items of a given type
     List {
-        #[arg(value_parser = parse_item)]
-        type_: ItemType,
+        #[arg(value_enum)]
+        /// Type of items to list
+        type_: ItemTypes,
     },
     /// Monitor registered items of a given type
     Monitor {
-        #[arg(value_parser = parse_item)]
-        type_: ItemType,
+        #[arg(value_enum)]
+        /// Type of items to monitor
+        type_: ItemTypes,
     },
     /// Show item properties
     Properties {
-        #[arg(value_parser = parse_item)]
-        type_: ItemType,
+        #[arg(value_enum)]
+        /// Type of items to show properties of
+        type_: ItemTypes,
```

Comment:
This is net progress so again not a merge blocker for me; But one thing that has annoyed me at times is that for the properties command the type should really be singular not plural (e.g. `properties devices 1`) makes no sense.  Though for properties it's possible to refactor into the item types specific commands at least and have (`device 1 properties`); monitor doesn't really fit into that mold though so we should keep that.